dev-compose.yaml

```diff
@@ -2,11 +2,13 @@ name: photoview
 
 services:
   dev-ui:
-    image: photoview/photoview-ui
+    image: photoview/ui
+    profiles:
+    - dev
```

Comment:
why do you need the profile if there are only 2 services defined and both have this profile set? You can do the same with the command `docker compose -f ./dev-compose.yaml <action>`

Reply:
Well, we may put more services in `dev-compose.yaml` in the future, like databases or test environments. With the giving profile, there's no need to change the command then.

Reply:
This is a good point to add to the documentation: the possibility for devs to extend this compose with additional services

Reply:
Removed dev profile as using profiles with database.